tpm/src/socket.rs

```diff
@@ -0,0 +1,139 @@
+// Copyright © 2022, Microsoft Corporation
+//
+// SPDX-License-Identifier: Apache-2.0 OR BSD-3-Clause
```

Comment:
Why does this file have "OR BSD-3-Clause"?

Reply:
Thanks for catching this, it was incorrectly captured. Fixed now.